src/components/seed-search-form.jsx

```diff
@@ -217,6 +226,7 @@ class SeedSearchForm extends React.Component {
           </div>
         </div>
         {this.seedTypeControls()}
+        <strong>Tip!</strong> Try a genre search by typing <a onClick={e => this.seedGenre(e)}>{genreValue}</a>
```

Comment:
Maybe `setGenreFilter` as a name. You could also pass `genreValue` to the function as the second parameter so you don't have to look at the innerHTML.